scheduler/server/job_state.go

```diff
@@ -20,6 +20,11 @@ type jobState struct {
 	JobKilled      bool         //indicates the job was killed
 	TimeCreated    time.Time    //when was this job first created
 	TimeMarker     time.Time    //when was this job last marked (i.e. for reporting purposes)
+
+	// track tasks by state (completed, running, not started) for new scheduling alg
```

Comment:
can we change this to indicate these fields are only used by the priority sched alg, and that this is a temporary workaround?

Reply:
done